best403unlocker

```diff
@@ -101,12 +109,17 @@ else
 fi
 
 }
-
 function download_speed() {
-du -s /tmp/$1 >> $DATABASE_PATH
-rm -rf /tmp/$1
+    local file="/tmp/$1"
+    if [[ -f $file ]]; then
```

Comment:
when we check an docker image it will store the downloaded files as a directory instead of regular file and this condition needs to support directory too

Reply:
این تیکه هم یکسری تغییرات کرد .هر دو موضوع رو بررسی میکند.